tx-pool/src/process.rs

```diff
@@ -969,11 +969,15 @@ impl TxPoolService {
         let total = txs.len();
         let mut count = 0usize;
         for tx in txs {
-            if self._process_tx(tx, None).await.is_err() {
+            let tx_hash = tx.hash();
+            if let Err(err) = self._process_tx(tx, None).await {
+                error!("failed to process {:#x}, error: {:?}", tx_hash, err);
```

Comment:
I think we'd better add a trace target to save the tx into a file. Example:

```
if log_enabled!(target: a_trace_traget_for_saveing_tx) {
  save_tx_into_file(&tx);
}
```

Reply:
I agree with it. We can save these errored transactions inside `fn _process_tx`.

In this PR, I just think it would be better to log the processing error outside of `self._process_tx()`.